extensions/wikia/Email/Controller/FounderDigestController.class.php

```diff
@@ -53,8 +53,8 @@ protected function assertValidWikiId() {
 	 * @throws \Email\Check
 	 */
 	protected function assertPageViewsSet() {
```

Comment:
This logic was a little off; checking for an empty value but not a stringified zero value.  My assumption was that this was supposed to let zero pages views generate an email, but this won't work for integer zero values.  Cristina and I agree we should just skip all zero page view emails.